tests/Unit/Controllers/Store/CentreControllerTest.php

```diff
@@ -208,7 +218,7 @@ public function testItCanDownloadASocialPrescriptionsRegistrationsSpreadsheet()
             "Eligible From",
             "Due Date",
         ];
-
+\Log::info($headers);
```

Comment:
Did you want to keep this here or was it an accident?

Reply:
😮 Well spotted!